var/spack/repos/builtin/packages/py-tensorflow/package.py

```diff
@@ -1009,9 +947,6 @@ def build(self, spec, prefix):
             # removed in bazel 3.6
             args.append("--incompatible_no_support_tools_in_action_inputs=false")
 
-        if spec.satisfies("@2.9: platform=darwin"):
-            args.append("--macos_sdk_version={}".format(macos_version()))
```

Comment:
This turned out to be the root cause of https://github.com/tensorflow/tensorflow/issues/57631